lib/node_modules/@stdlib/blas/base/sscal/benchmark/c/Makefile

```diff
@@ -0,0 +1,110 @@
+#/
```

Comment:
This is an "older" version of this file. Use [`ddot`](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/blas/base/ddot/benchmark/c/Makefile) as a reference.

Reply:
Okay - I'll keep that in mind for the future